hysds/configs/celeryconfig_local.py

```diff
@@ -1,5 +1,5 @@
-broker_url = "amqp://guest:guest@rabbitmq:5672//"
-result_backend = "redis://redis:6379"
+broker_url = "amqp://guest:guest@rabbitmq:${rabbitmq_service_port}//"
```

Comment:
Might have missed something here, but do we want to assume the `guest` user here for all cases? Or make that configurable?

Reply:
@riverma: yes that can also probably be extracted out to a terraform configuration but I'll leave that to the U-SPS team. This PR only provides updates to be able to deploy SPS generically in non-Unity MCP accounts.